src/js/tabs/osd.js

```diff
@@ -2401,30 +2403,38 @@ TABS.osd.initialize = function (callback) {
         // init structs once, also clears current font
         FONT.initData();
 
+        // Some of these definitions are determined by version.
+        SYM.AH_CENTER_LINE = 0x26;
+        SYM.AH_CENTER_LINE_RIGHT = 0x27;
+        if(semver.gte(CONFIG.apiVersion, "1.42.0")) {
+            SYM.AH_CENTER_LINE = 0x7B;
+            SYM.AH_CENTER_LINE_RIGHT = 0x7D;
+        }
+
```

Comment:
I'm not sure if  I should put a comment near the top of this where all the rest of SYM is defined, mentioning uh, wherever in the code this is. like `// some of these are changed in the initialization function below`

or maybe move all of the definitions down here

Reply:
Good point, I think so.

Reply:
Going with the comment at the top. Otherwise I'd have to give some consideration to the `var STICK_OVERLAY_SPRITE` array